hangman.py

```diff
@@ -0,0 +1,85 @@
+#importing the time module
+import time
+
+#welcoming the user
+name = input("What is your name? ")
+
+print ("Hello"   +name + " Time to play hangman!")
+
+print (" ")
+
+#wait for 1 second
+time.sleep(1)
+
+print("Start guessing...")
+time.sleep(0.5)
+
+#here we set the hello
+word = "hello"
+
+#creates an variable with an empty value
+guesses = ''
+
+#determine the number of turns
+turns = 10
+
+# Create a while loop
+
+#check if the turns are more than zero
+while turns > 0:
+
+    # make a counter that starts with zero
+    failed = 0
+
+    # for every character in hello_word
+    for char in word:
+
+    # see if the character is in the players guess
+        if char in guesses:
+
+        # print then out the character
+            print (char)

```

Comment:
In this position for example **hello** when I type the 4th letter _l_ 
like => h e l _ _
it automatically takes the next letter l as well.
![image](https://user-images.githubusercontent.com/65100038/193414966-324ab7bd-cacb-404a-bd03-fac5c8086f5f.png)

try to fix this if this is possible